app/client/src/sagas/WidgetBlueprintSagas.ts

```
@@ -110,12 +112,27 @@ export function* executeWidgetBlueprintOperations(
+      case BlueprintOperationTypes.ADD_ACTION:
+        if (operation.fn) {
+          const updatePropertyPayloads: UpdatePropertyArgs[] | undefined =
+            yield (operation.fn as BlueprintOperationAddActionFn)(
+              widget as WidgetProps & { children?: WidgetProps[] },
+            );
+
+          updatePropertyPayloads &&
+            updatePropertyPayloads.forEach((params: UpdatePropertyArgs) => {
+              widgets[params.widgetId][params.propertyName] =
+                params.propertyValue;
+            });
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)